Sources/ApolloCodegenLib/Templates/RenderingHelpers/GraphQLType+Rendered.swift

```diff
@@ -173,4 +177,47 @@ extension GraphQLNamedType {
 
     return swiftName
   }
+
+  fileprivate func qualifiedRootTypeName(
+    in context: GraphQLType.RenderContext,
+    replacingNamedTypeWith newTypeName: String? = nil,
+    config: ApolloCodegenConfiguration
+  ) -> String {
+    lazy var typeName = { newTypeName ?? self.swiftName.firstUppercased }()
```

Comment:
_This doesn't need to change, my question is more out of curiosity._

 Is there a benefit to `typeName` and `schemaModuleName` being defined as `lazy`? The return references both of them; it doesn't look like there is a logic path where they wouldn't be used.